src/pages/Dashboard/Automations/AddAction.vue

```diff
@@ -320,19 +334,33 @@ export default {
                 }
               }
               break
-            case 'PAGERDUTY': {
-              this.actionConfig = {
-                api_token_secret: this.apiToken,
-                routing_key: this.routingKey,
-                severity: this.severity
+            case 'PAGERDUTY':
+              {
+                this.actionConfig = {
+                  api_token_secret: this.apiToken,
+                  routing_key: this.routingKey,
+                  severity: this.severity
+                }
+                if (this.messageText) {
+                  //TO DO - Don't think this is working - need to fix!
+                  this.actionConfig.message = this.bothMessages
+                    ? `{} ${this.messageText}`
+                    : this.messageText
+                }
```

Comment:
do we want to address this before merging?

Reply:
Oh hm I didn't write this so I'm not sure let me look